var/spack/repos/builtin/packages/py-bluepy/package.py

```diff
@@ -80,3 +81,8 @@ def patch(self):
         filter_file("'jsonschema>=2.3.0',", "", "setup.py")
         filter_file("'progressbar2>=3.18',", "", "setup.py")
         filter_file("'shapely>=1.3.2',", "", "setup.py")
+
+    @run_after('install')
+    @on_package_attributes(run_tests=True)
+    def pip_check(self):
+        python('-mpip', 'check')
```

Comment:
We also have this check here: https://github.com/BlueBrain/spack/blob/347d9a37e51fbd2de61d5aaf71c269159175119c/lib/spack/spack/build_systems/python.py#L356

Relies only on `setup.py` rather than `pip`… makes me wonder if we can find some way to hook that up automatically (and why that was not done before)

Reply:
It seems to me that it would run `python setup.py check` that would check only metadata and restructuredtext, not dependencies, according to
https://github.com/pypa/setuptools/blob/c121d289da5d19cf6df2bf6b64ac28916a060161/setuptools/_distutils/command/check.py#L63-L77

Reply:
Bummer.